Unable to serve assets when running server locally

I'm currently trying to run snikket-web-portal locally to integrate it in a Dockerfile. I'm following the instructions to run the dev server, however I encountered this error message when running `make`: 
```
python3 -m scss --load-path snikket_web/scss/ -o "snikket_web/static/css/app.css" "snikket_web/scss/app.scss"
[…]
FileNotFoundError: [Errno 2] No such file or directory: 'snikket_web/static/css/app.css'
make: *** [Makefile:16: snikket_web/static/css/app.css] Error 1
```

To let Make work, I had to do `touch snikket_web/static/css/app.css`

However, after running the server, I see the assets are not served correctly:
```
[2020-04-28 17:04:36,130] Running on 0.0.0.0:5000 over http (CTRL + C to quit)
[2020-04-28 17:04:38,745] 172.17.0.1:33502 GET /login 1.1 200 1156 24325
[2020-04-28 17:04:38,905] 172.17.0.1:33502 GET /static/css/app.css 1.1 404 103 4128
[2020-04-28 17:04:38,911] 172.17.0.1:33504 GET /static/css/common.css 1.1 404 103 7325
[2020-04-28 17:04:38,933] 172.17.0.1:33502 GET /favicon.ico 1.1 404 103 3212
[2020-04-28 17:04:39,762] 172.17.0.1:33502 GET /service-worker.js 1.1 404 103 2035
[2020-04-28 17:04:40,359] 172.17.0.1:33510 GET /login 1.1 200 1156 2939
[2020-04-28 17:04:40,441] 172.17.0.1:33510 GET /static/css/app.css 1.1 404 103 3154
[2020-04-28 17:04:40,448] 172.17.0.1:33514 GET /static/css/common.css 1.1 404 103 4843
[2020-04-28 17:04:40,459] 172.17.0.1:33514 GET /favicon.ico 1.1 404 103 1833
```
In the container I see not assets have been built:
```
# ls  snikket_web/static/css/
# 
```

Am I missing on something in the Dockerfile? Or is something missing in the Makefile?

Here is the work-in-progress Dockerfile I was trying:
```
FROM python:3.7-slim-buster

ENV DEBIAN_FRONTEND noninteractive

RUN apt-get update \
    && apt-get install -y --no-install-recommends \
    make \
    && apt-get clean

COPY Makefile /opt/snikket-web-portal/Makefile

COPY requirements.txt /opt/snikket-web-portal/requirements.txt

COPY build-requirements.txt /opt/snikket-web-portal/build-requirements.txt

COPY snikket_web/ /opt/snikket-web-portal/snikket_web

COPY web_config.py /opt/snikket-web-portal/web_config.py

COPY babel.cfg /opt/snikket-web-portal/babel.cfg

WORKDIR /opt/snikket-web-portal

RUN pip install -r requirements.txt \
    && pip install -r build-requirements.txt

RUN mkdir -p snikket_web/static/css \
    && make

ENV QUART_APP snikket_web:app
ENV QUART_ENV production
ENV SNIKKET_WEB_CONFIG "/opt/snikket-web-portal/web_config.py"

# For production
#RUN pip install hypercorn
#CMD ["hypercorn", "-b", "0.0.0.0:8000", "snikket_web:app"]

# For testing purpose
CMD ["quart", "run", "-h", "0.0.0.0"]
```

--- a/Makefile
+++ b/Makefile
@@ -13,6 +13,7 @@ all: build_css compile_translations
 build_css: $(generated_css_files)
 
 $(generated_css_files): snikket_web/static/css/%.css: snikket_web/scss/%.scss $(scss_includes)
+	mkdir -p snikket_web/static/css/
 	$(SCSSC) -o "$@" "$<"
 
 clean:

